neurodamus/core/coreneuron_configuration.py

```diff
@@ -0,0 +1,117 @@
+import os
+import logging
+from ._utils import run_only_rank0
+from . import NeurodamusCore as Nd
+
+class _CoreConfig(object):
```

Comment:
I find that name a wee bit ambiguous: the core of what? `CoreNeuronConfig` maybe?